### PR TITLE
feat(confirmsendmodal): show input if invoice amount is 0 #127

### PR DESCRIPTION
**Pull request by escapedcat**
- [x] fix failing SendOnChain "validates amount is lower than balance"-test
- [x] add tests to ConfirmSendModal

### CODE REVIEW

File: src/components/Shared/SendModal/ConfirmSendModal/ConfirmSendModal.test.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      await waitFor(() => expect(amountInput).not.toHaveClass("input-error"));
+
+      expect(
+        screen.getByRole("button", { name: "check.svg settings.confirm" })
+      ).not.toBeDisabled();
+    });
+  });
+
+  describe("ln-invoice with amount above zero", () => {
+    test("valid form passes", async () => {
```
Comment (escapedcat) [inline src/components/Shared/SendModal/ConfirmSendModal/ConfirmSendModal.test.tsx:85]: This test is passing but throwing an `act`-error:  

```sh
  console.error
    Warning: An update to ConfirmSendModal inside a test was not wrapped in act(...).

    When testing, code that causes React state updates should be wrapped into act(...):

    act(() => {
      /* fire events that update state */
    });
    /* assert on the output */

    This ensures that you're testing the behavior the user would see in the browser. Learn more at https://reactjs.org/link/wrap-tests-with-act
        at ConfirmSendModal (/raspiblitz-web/src/components/Shared/SendModal/ConfirmSendModal/ConfirmSendModal.tsx:13:17)
        at I18nextProvider (/raspiblitz-web/node_modules/react-i18next/dist/commonjs/I18nextProvider.js:13:19)
```
FML, no idea why or where to add the `act`. Tried several ways.

**Comment by cstenglein** [comment]
Cool thing! Thank you! :)

Will look on it on Friday after Aarons PR (Thursday will be more work on React Router 6) :)

File: src/components/Shared/SendModal/ConfirmSendModal/ConfirmSendModal.test.tsx (hunk 1 above)
Comment (escapedcat) [reply src/components/Shared/SendModal/ConfirmSendModal/ConfirmSendModal.test.tsx:85]: Found it...

**Review by cstenglein — APPROVED** [review]
Works, thank you! :)

**Pull request merged**